src/Whim.LayoutPreview/LayoutPreviewPlugin.cs

```diff
@@ -86,6 +87,12 @@ private void WindowMoved(object? sender, WindowMoveEventArgs e)
 
 			DraggedWindow = e.Window;
 			ILayoutEngine layoutEngine = workspace.ActiveLayoutEngine.MoveWindowToPoint(e.Window, normalizedPoint);
+			if (layoutEngine.GetLayoutEngine<FreeLayoutEngine>() is not null)
+			{
+				// To be renamed when FreeLayoutEngine will be renamed
+				Logger.Debug("Skip LayoutPreview as LeafLayoutEngine is a FreeLayoutEngine");
+				return;
+			}
```

Comment:
It is not really testable.
I could do a "pseudo" test just to have coverage, but it is not really meaningful as a test.

Reply:
It's fine - I haven't tested a bunch of code which requires a Windows STA thread.